client/templates/reporting/court-reports.njk

```diff
@@ -45,7 +45,7 @@
       <h2 class="govuk-heading-m">Senior jury officer reports</h2>
       <ul class="govuk-list">
         <li><a class="govuk-link" href="#">Jury ammendment report</a></li>
-        <li><a class="govuk-link" href="#">Manually-created jurors report</a></li>
+        <li><a class="govuk-link" href="{{ url('reports.manual-juror-report.filter.get') }}">Manually-created jurors report</a></li>
       </ul>
```

Comment:
we should hide these links behind `{% isSJO %}`

Reply:
Done